adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboost.m

```diff
@@ -104,10 +116,20 @@ - (void)presentInterstitialFromRootViewController:(UIViewController *)rootViewCo
 
 #pragma mark Banner
 
-- (void)getBannerWithSize:(GADAdSize)adSize {
-  // Chartboost doesn't support banner ads.
-  NSError *error = GADChartboostErrorWithDescription(@"Chartboost Ads doesn't support banner ads.");
-  [_interstitialConnector adapter:self didFailAd:error];
+- (void)getBannerWithSize:(GADAdSize)adSize
+{
+    __weak typeof(self) weakSelf = self;
+    [self initializeChartboost:^(BOOL success) {
+        if (success) {
+            GADMAdapterChartboost *strongSelf = weakSelf;
+            UIViewController *viewController = [strongSelf->_interstitialConnector viewControllerForPresentingModalView];
```

Comment:
I think we can rename `_interstitialConnector` to just `_connector` since it's being used for Banners now.